src/apps/components/AppAlerts/useSidebarWebhookAlertMetadata.ts

```diff
@@ -0,0 +1,57 @@
+import { useUser } from "@dashboard/auth";
+import { MetadataInput, useUserAccountUpdateMutation } from "@dashboard/graphql";
+import { useMemo } from "react";
+
+const deliveryAttemptKey = "sidebar_app_webhook_alert_state";
```

Comment:
suggestion: we can export it, or put into separate file so that this string can be re-used for tests

```suggestion
export const DELIVERY_ATTEMPT_KEY = "sidebar_app_webhook_alert_state";
```